src/main/java/ja/workshops/spring/części/konfiguracja/programistyczna/c_specjalistyczne_ziarna/KreatywnyKontroler.java

```diff
@@ -0,0 +1,48 @@
+package ja.workshops.spring.części.konfiguracja.programistyczna.c_specjalistyczne_ziarna;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+/**
+ * Kontroler - w aplikacjach webowych bramka wejściowa do aplikacji.
+ * Pobiera dane od użytkownika, oraz zwraca odpowiedni widok - HTML.
+ *
+ * <p><i>W imię prezentacji HTML jest zaszyty w zmiennej. Na projekcie tak nie róbcie ;(</i></p>
+ * <br/>
+ *
+ * Q: What did the server say to his client who was having a bad day?
+ * <br/>...<br/>
+ * A: Everything's going to be 200
+ *
+ * @author Wojciech Makiela
+ */
+@Controller
+class KreatywnyKontroler {
+
+    private static final String HTML = "<!DOCTYPE html>\n" +
```

Comment:
Część mnie się spodziewała - w imię konwencji z dokumentacji - że to `złyHTML` będzie :-)